src/shadowbox/server/server_access_key.ts

```diff
@@ -53,8 +53,14 @@ class ServerAccessKey implements AccessKey {
   name: string;
   metricsId: AccessKeyMetricsId;
   readonly proxyParams: ProxyParams;
-  quota?: AccessKeyQuota;
-  isOverQuota?: boolean;
+  quotaUsage?: AccessKeyQuotaUsage;
+}
+
+export function IsAccessKeyOverQuota(accessKey: AccessKey) {
```

Comment:
You could make this an `AccessKey.IsOverQuota()` method

Reply:
Done in 24473f3.